rotkehlchen/db/dbhandler.py

```diff
@@ -2297,6 +2357,9 @@ def query_timed_balances(
                     ),
                 )
 
+        inferred_balances = self._infer_zero_timed_balances(cursor, balances, from_ts, to_ts)
+        balances.extend(inferred_balances)
```

Comment:
question: this doesn't guarantee that they are sorted by timestamp rigth? is that a problem?